package.json

```diff
@@ -63,19 +63,22 @@
 		"clean": "rimraf dist node_modules coverage"
 	},
 	"devDependencies": {
-		"@types/jest": "^29.5.2",
-		"@typescript-eslint/eslint-plugin": "5.60.1",
-		"@typescript-eslint/parser": "5.60.1",
-		"eslint": "^8.43.0",
-		"eslint-plugin-prettierx": "github:utily/eslint-plugin-prettierx#utily-20231004",
-		"eslint-plugin-simple-import-sort": "^10.0.0",
-		"jest": "^29.5.0",
-		"prettierx": "github:utily/prettierx#utily-20231004",
-		"rimraf": "^5.0.1",
-		"ts-jest": "^29.1.0",
-		"typescript": "^5.1.3"
+		"@types/jest": "^29.5.12",
+		"@typescript-eslint/eslint-plugin": "6.17.0",
+		"@typescript-eslint/parser": "6.17.0",
+		"eslint": "^8.56.0",
+		"eslint-plugin-prettierx": "github:utily/eslint-plugin-prettierx#utily-20240516",
+		"eslint-plugin-simple-import-sort": "^12.1.0",
+		"jest": "^29.7.0",
+		"prettierx": "github:utily/prettierx#utily-20240516",
+		"rimraf": "^5.0.7",
+		"ts-jest": "^29.1.4",
+		"typescript": "^5.4.5"
 	},
 	"overrides": {
```

Comment:
no longer need to override this i think.

Reply:
it is indeed no longer required

Reply:
Removed